test/ui/PipelineContextTest.ts

```diff
@@ -54,18 +54,75 @@ export function PipelineContextText() {
         it('should return the context value', () => {
           expect(test.cmp.getContextValue('qwerty')).toBe('azerty');
         });
+
+        it('should support setting a new context', () => {
+          test.cmp.setContext({ abc: 'def', hij: ['klm', 'nop'] });
+          expect(test.cmp.getContextKeys()).toEqual(jasmine.arrayContaining(['abc', 'hij']));
+        });
+
+        it('should support setting a single context value', () => {
+          test.cmp.setContextValue('popo', 'qwerty');
+          test.cmp.setContextValue('123', ['456', '789']);
+          expect(test.cmp.getContextValue('popo')).toEqual('qwerty');
+          expect(test.cmp.getContextValue('123')).toEqual(['456', '789']);
+        });
+
+        describe('with a global Coveo.context variable', () => {
+          beforeEach(() => {
+            Coveo['context'] = {};
+            Coveo['context']['productName'] = 'ACME 2000 ULTIMATE EDITION';
+            Coveo['context']['userRole'] = 'SYS ADMIN';
+          });
+
+          afterEach(() => {
```

Comment:
Since the `beforeEach` is setting `Coveo['context']` to an empty object, is the `afterEach` necessary?